client/js/ace/mode-fbp.js

```diff
@@ -766,6 +767,45 @@ oop.inherits(Mode, TextMode);
 
     this.$id = "ace/mode/fbp";
 }).call(Mode.prototype);
+// adding some clear formatting
+var fbpcompletion=function(){
+
+};
```

Comment:
Why do we have an empty function here ? Also try putting space between them it like tihs:

`var fbpcompletion = function() {`